Neos.Flow/Classes/Http/Helper/ResponseInformationHelper.php

```diff
@@ -198,11 +198,9 @@ public static function makeStandardsCompliant(ResponseInterface $response, Reque
             $response = $response->withBody(stream_for(''));
         }
 
-        $cacheControlHeaderLine = $response->getHeaderLine('Cache-Control');
-
-        if ((!empty($cacheControlHeaderLine) && strpos('no-cache', $cacheControlHeaderLine) !== false)
+        $cacheControlHeaderValue = $response->getHeaderLine('Cache-Control');
+        if (($cacheControlHeaderValue !== '' && strpos('no-cache', $cacheControlHeaderValue) !== false)
```

Comment:
Now that I look at this - the not empty check made me suspicious - `strpos` is "haystack, needle", so shouldn't that be `strpos($cacheControlHeaderValue, 'no-cache')` and then the empty check isn't needed any more? See https://3v4l.org/gOJHp#v7.4.25